governance/program/src/error.rs

```diff
@@ -393,6 +393,14 @@ pub enum GovernanceError {
     /// GovernanceConfig change not allowed
     #[error("GovernanceConfig change not allowed")]
     GovernanceConfigChangeNotAllowed,
+
+    /// All vote thresholds can't be disabled
+    #[error("All vote thresholds can't be disabled")]
+    AllVoteThresholdsDisabled,
```

Comment:
nit: `AllVoteThresholdsDisabled` means something a bit different than `all vote thresholds can't be disabled`, how about `AtLeastOneVoteThresholdRequired`?

Reply:
done